bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/MetadataBookieDriver.java

```diff
@@ -68,6 +71,29 @@ LedgerManagerFactory getLedgerManagerFactory()
      */
     LayoutManager getLayoutManager();
 
+    /**
+     * Return health check is enable or disable.
+     *
+     * @return true if health check is enable, otherwise false.
+     */
+    default CompletableFuture<Boolean>  isHealthCheckEnabled() {
+        return FutureUtils.value(true);
+    }
+
+    /**
+     * Disable health check.
+     */
+    default CompletableFuture<Void> disableHealthCheck() throws Exception {
```

Comment:
methods that return CompletableFuture should not throw Exceptions.

you have to return a failed CompletableFuture

Reply:
like this?